css/style.css

```diff
@@ -9,3 +9,126 @@ header {
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
 }
+
+body {
+	background-color: #f0efef;
+	font-family: 'Open Sans',sans-serif;
+	margin: 0 auto;
+}
+
+h1,h2,p {
+	color: #FFFFFF;
+}
+
+h1 {
+	padding-top: 20px;
+}
+
+h1,ul,p.b1 {
+	padding-left: 20px;
+}
+
+h2 {
+ 	font-weight; 300;
+ 	text-align: left;
+ 	padding-top: 200px;
+ 	padding-bottom: 50px;
+}
+
+h3 {
+	font-size: 20px;
+	font-weight: 400;
+	color: #121212;
+	padding-left: 20px;
+	padding-top: 40px;
+	padding-bottom: 20px;
+}
+
+p.b1 {
+	color: #606161;
+}
+
+.benefits {
+	background-color: #f0efef;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	padding-left: 200px;
+	padding-right: 200px;
+}
+
+ul {
+	list-style-position: inside;
+	margin: 0 auto;
+	padding-top: 10px;
+	color: #606161;
+	font-weight: 300;
+	font-size: 13px;
+}
+
+li {
+	padding-left: 3px;
+	font-size: 13px;
+}
+
+.testimonial {
+	background: url("../images/testimonial_bg.jpg") no-repeat;
+	height: 50px;
```

Comment:
If you remove this height the size of the element will auto adjust to the content.